Sem. 03/Pract. 03/Task01.cpp

```diff
@@ -0,0 +1,197 @@
+#include <iostream>
```

Comment:
check https://github.com/GeorgiTerziev02/Object-oriented_programming_FMI/pull/24 for comments. Note that you shouldn't make two different pull requests for the same task. Remove the file Task01.cpp from this pull request so there are no merge confilicts.